src/js/themes/base.js

```diff
@@ -880,6 +880,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       },
       check: {
         radius: '100%',
+        background: {
+          // color: undefined,
+        },
```

Comment:
@ShimiSun from a convention standpoint, do you want the entire `background` property commented out since none of the object properties are defined?

```
        // background: {
          // color: undefined,
        // },
```

Reply:
Yes, I agree it should be commented out.

Reply:
Makes sense, thanks.

Reply:
If we comment the entire `background` this line will throw an error because background will be `undefined`
https://github.com/grommet/grommet/blob/1a7e69e8406342456152dc4e840e9a20faea7959/src/js/components/RadioButton/StyledRadioButton.js#L63

So, we have to add some condition, e.g.:
```js
 background-color: ${props =>
    props.theme.radioButton.check.background &&
    props.theme.radioButton.check.background.color};
```

Do you want to follow this path?

Reply:
Good call. That seems reasonable to me and would be consistent with approaches elsewhere in Grommet.